kbs/config/docker-compose/kbs-config.toml

```diff
@@ -1,9 +1,18 @@
+[http_server]
 sockets = ["0.0.0.0:8080"]
-auth_public_key = "/opt/confidential-containers/kbs/user-keys/public.pub"
 insecure_http = true
 
-[attestation_token_config]
+[attestation_token]
```

Comment:
I think the changes to the configuration files should be reported in the various readme files as well (it seems they're not there yet).

Reply:
Yes, let me add the documents

Reply:
Thank you @Xynnn007. Can you please update doc/examples for the following as well?
- [http-server]
- [attestation_service]
- [admin]
- [plugins]
- [repository]
- [policy-engine]
I might have not included them all.

Or are you thinking to do it later? (e.g. later on in the review/other PR)

Reply:
Yes. I should do this. Thanks for the great suggestion.

Reply:
please check https://github.com/Xynnn007/kbs/blob/refactor-kbs/kbs/docs/config.md which is the view of current PR